generate/generate.go

```diff
@@ -334,6 +334,17 @@ func (g *Generator) ClearProcessEnv() {
 // AddProcessEnv adds env into g.spec.Process.Env.
 func (g *Generator) AddProcessEnv(env string) {
```

Comment:
I'd rather [take explict key and value args][1], which also avoids the “what happens if `env` doesn't contain an equal sign?” case.

[1]: https://github.com/opencontainers/runtime-tools/issues/290#issuecomment-267103862